# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

FindMarkerPoseActionGoal action_goal
FindMarkerPoseActionResult action_result
FindMarkerPoseActionFeedback action_feedback
